Gems/EMotionFX/Code/Source/Editor/Plugins/Ragdoll/RagdollNodeWidget.cpp

```diff
@@ -116,7 +116,8 @@ namespace EMotionFX
 
     void RagdollNodeWidget::InternalReinit()
     {
-        if (m_selectedModelIndices.size() == 1)
+        const QModelIndexList& selectedModelIndices = GetSelectedModelIndices();
```

Comment:
This is not safe. GetSelectedModelIndices() is returning a copy now. The returned copy might get destructed after this line and the reference might be invalid. Can you remove the ref here?

Reply:
Ah ok, good to know. I wasn't sure that all compilers are safe using that. Just shipped, still would prefer to just hold a copy in this case as the copy will be hold by the compiler internally anyway to make it safe while it seems kind of hidden when looking at the code that it is returning a copy.